src/firmware-update.html

```diff
@@ -3,30 +3,50 @@
-    <p>We're working on making firmware updates available via Home Assistant. You will receive updates automatically in the future.
+    <p>The Connect&nbsp;ZBT-1 ships with a full featured Zigbee firmware. <em>At this point, there are no known issues with the factory flashed firmware.</em></p>
+    <p>
+      The Nabu Casa team works on integrating firmware updates for Connect&nbsp;ZBT-1 straight from Home Assistant! There will be update notifications for
+      Connect&nbsp;ZBT-1 just like any other updates supported Home Assistant! If you have a working Zigbee or Thread setup today, please wait until the new
+      update system lands sometime early 2025.
     </p>
   </div>
-  <div><h3 class="text-left">SL Web Tools to flash firmware updates from the browser</h3></div>
-  <p>
-    This page is for troubleshooting. If you use Connect&nbsp;ZBT-1 in Home Assistant, the Home Assistant discovery flow will automatically install the correct firmware. You do not need to install firmware using this tool. Furthermore, on every start of the OpenThread Border router, the latest Thread firmware which is bundled with the add-on will be installed.
-  </p>
-  <p>
-    If for some reason you like to flash a firmware still: you can use the web-flasher below to install the latest firmware version for Home Assistant Connect&nbsp;ZBT-1 directly from your browser over USB.
-  </p>
+  <div>
+    <h3 class="text-left">Reflashing the Connect ZBT-1</h3>
+    <p>
+      Note: <a href="/faq/#what-is-the-difference-between-home-assistant-skyconnect-and-connect-zbt-1" target="_blank">Home Assistant SkyConnect</a> and Connect&nbsp;ZBT-1 use the same firmware. You can use the same firmware update process for both devices.
+    </p>
+    <p>If you must reflash your ZBT-1 today, the following options are available:</p>
+    <ul>
+      <li><b>For Thread:</b> The <a class="my"
+        href="https://my.home-assistant.io/redirect/supervisor_addon/?addon=core_openthread_border_router"
+        target="_blank"><b>OpenThread Border Router</b></a> add-on flashes the Thread firmware on startup. You can verify the add-on logs to see the output of the
+        firmware check or potential firmware update.</li>
+      <li><b>For Zigbee:</b> You can use the <a class="my"
+        href="https://my.home-assistant.io/redirect/supervisor_addon/?addon=core_silabs_flasher"
+        target="_blank"><b>Silicon Labs Flasher</b></a> add-on. The add-on will flash the Connect&nbsp;ZBT-1 and stop right after. Before starting the add-on:
+        <ul>
+          <li>Disable the <a class="my"
+            href="https://my.home-assistant.io/redirect/integration/?domain=zha"
+            target="_blank"><b>Zigbee Home Automation</b></a> integration or any add-on which might use the Connect&nbsp;ZBT-1 at point of flashing</li>
+          <li>Start the add-on</li>
+        </ul>
+        Verify the add-on logs to verify successful flashing.
```

Comment:
should this be part of the unordered list above?

Reply:
It is meant to be still part of the Zigbee firmware. The Thread firmware above has it's own (similar) sentence.